### PR TITLE
Fixed first message chat creation

### PR DESCRIPTION
**Pull request by micheledallerive**
The chat was not successfully creating when sending a non-text first message.

### CODE REVIEW

**Comment by JekxDevil** [comment]
Issue #89 fixed by this pull request.

**Review by JekxDevil — APPROVED** [review]
good handling in api.

File: client/src/components/ChatBoard.vue [hunk 1]
```diff
@@ -122,12 +122,6 @@ export default {
     },
     async sendMessage() {
       if (this.onlySpaces(this.messageString)) return;
-      let chatId = this.$props.chat._id;
-      // if the chat does not exist we create a new one and get the save the id
-      if (chatId === null) {
-        const { data } = await this.$api.createChat(this.chat.members[0]._id);
-        chatId = data._id; // way to unpack data apparently
```
Comment (DonnaDona) [inline client/src/components/ChatBoard.vue:129]: :(

**Review by DonnaDona — APPROVED** [review]

File: client/src/components/ChatBoard.vue (hunk 1 above)
Comment (JekxDevil) [reply client/src/components/ChatBoard.vue:129]: :(

**Pull request merged**